concent-vm/install-golem.yml

```diff
@@ -0,0 +1,54 @@
+- hosts:
+    - vagrant
+  vars_files:
+    - ../concent-builder/repositories.yml
+    - vars.yml
+  tasks:
+    - become:      yes
+      become_user: vagrant
+      block:
+        - name:   Clone golem repository
+          git:
+            repo:    "{{ repositories.golem.url }}"
+            dest:    "{{ golem_dir }}"
+            version: "{{ golem_version }}"
+            clone:   yes
+            update:  yes
+
+        - name:   Create virtualenv for golem
+          command:  virtualenv ~/golem_virtualenv --python python3.6 creates="~/golem_virtualenv"
```

Comment:
Ansible's [pip](https://docs.ansible.com/ansible/latest/modules/pip_module.html) command can create a virtualenv automatically if it does not exist.